<?xml version="1.0" encoding="utf-8"?>
<launch>
  <arg name="kinect2_nodelet_manager" default="kinect2"/>

  <node pkg="pses_kinect_filter" type="kinect_filter" name="kinect_filter" output="screen">
	<param name="depth_image_topic" type="str" value="/kinect2/sd/image_depth_rect" />
	<param name="output_depth_image_topic" type="str" value="/kinect_filter/depth_image" />
	<param name="camera_info_topic" type="str" value="/kinect2/sd/camera_info" />
  </node>

  <node pkg="nodelet" type="nodelet" name="kinect2_points"
	args="load depth_image_proc/point_cloud_xyz $(arg kinect2_nodelet_manager)" respawn="true" output="screen">
    <remap from="image_rect"        to="kinect_filter/depth_image"/>
    <remap from="camera_info"       to="kinect_filter/camera_info"/>
    <remap from="points"            to="kinect_filter/points"/>	
  </node>

</launch>
